backend/src/src-fossology/src/main/java/org/eclipse/sw360/fossology/FossologyHandler.java

```diff
@@ -162,12 +164,14 @@ public ExternalToolProcess process(String releaseId, User user, String uploadDes
             handleUploadStep(componentClient, release, user, fossologyProcess, sourceAttachment, uploadDescription);
         } else if (FossologyUtils.FOSSOLOGY_STEP_NAME_SCAN.equals(furthestStep.getStepName())) {
             handleScanStep(componentClient, release, user, fossologyProcess);
-        } else if (FossologyUtils.FOSSOLOGY_STEP_NAME_REPORT.equals(furthestStep.getStepName())) {
-            handleReportStep(componentClient, release, user, fossologyProcess);
         }
-
         updateFossologyProcessInRelease(fossologyProcess, release, user, componentClient);
 
+        if(BackendUtils.DISABLE_CLEARING_FOSSOLOGY_REPORT_DOWNLOAD && FossologyUtils.FOSSOLOGY_STEP_NAME_REPORT.equals(furthestStep.getStepName())) {
```

Comment:
Should it be the **not** case for `BackendUtils.DISABLE_CLEARING_FOSSOLOGY_REPORT_DOWNLOAD` as this step should run only if the configuration is set to `false`?

Reply:
made the changes